extensions/amp-story/0.1/media-pool.js

```diff
@@ -575,8 +594,17 @@ export class MediaPool {
     const isMuted = mediaEl.muted;
     const currentTime = mediaEl.currentTime;
 
-    mediaEl.muted = false;
-    return Promise.resolve(mediaEl.play()).then(() => {
+    // If the video is already playing, we do not want to call play again, as it
+    // can interrupt the video playback.  Instead, we do a no-op.
+    const playFn = () => {
+      if (isPaused) {
+        mediaEl.play();
+      }
+    };
+
+    return Promise.resolve(playFn()).then(() => {
```

Comment:
This invokes `playFn` synchronously. Did you mean to do this instead?

```js
return Promise.resolve().then(() => playFn()).then(() => {
```

Reply:
Playing it synchronously works, but we can also do this.

Reply:
If synchronous play works, then I'd just remove the function and call directly. Then use:

```js
return Promise.resolve().then(() => {
```